### PR TITLE
Update to Trino: The Definitive Guide

### PR DESCRIPTION
**Pull request by mosabua**
* Update all content to use the new name
* Add redirector in netlify config (tested on deployment here and locally)
* Update page name
* Update image (leaving old one in place for now)
* Update all incoming links and references
* Update content of page very slightly (companies using Trino mostly)

Would be great to get this live before the meetup tomorrow. 

Also .. I will follow up with a blog post once the free download from Starburst sponsorship is updated. I will also update the links then (including change to starburst.io)

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -21,3 +21,7 @@
 [[redirects]]
   from = "/star"
   to = "https://github.com/trinodb/trino/blob/master/.github/star.png"
+
+[[redirects]]
+  from = "/presto-the-definitive-guide"
```
Comment (electrum) [inline netlify.toml:26]: This doesn't work for `presto-the-definitive-guide.html` which is how it's linked today. We should redirect both. You can test with the "deploy-preview" CI check that has a full hosted version of the preview site.

File: netlify.toml [hunk 2]
```diff
@@ -21,3 +21,7 @@
...
+  to = "/trino-the-definitive-guide"
```
Comment (electrum) [inline netlify.toml:27]: Nit: add newline

**Review by electrum — COMMENTED** [review]
One comment about redirect, otherwise looks good

File: netlify.toml (hunk 1 above)
Comment (mosabua) [reply netlify.toml:26]: Added both .. will test in a sec

File: netlify.toml (hunk 2 above)
Comment (mosabua) [reply netlify.toml]: added

Comment (mosabua) [reply netlify.toml]: Fixed

File: netlify.toml (hunk 1 above)
Comment (mosabua) [reply netlify.toml:26]: Both redirects are in place now and work.

**Comment by mosabua** [comment]
Thanks @martint !

**Pull request merged**